app/form/v1_0/fields/drug-use/drug-use-analysis.ts

```diff
@@ -0,0 +1,19 @@
+import FormWizard from 'hmpo-form-wizard'
+import { FieldType, ValidationType } from '../../../../../server/@types/hmpo-form-wizard/enums'
+
+const drugsPractitionerAnaLysisMotivatedToStop: FormWizard.Field = {
+  text: `Does [subject] seem motivated to stop or reduce their drug use?`,
+  code: `drugs_practitioner_analysis_motivated_to_stop`,
+  type: FieldType.Radio,
+  validate: [{ type: ValidationType.Required, message: 'Select if there are any strengths or protective factors' }],
+  options: [
+    { text: 'Does not show motivation to stop or reduce', value: 'NO_MOTIVATION', kind: 'option' },
+    { text: 'Shows some motivation to stop or reduce ', value: 'PARTIAL_MOTIVATION', kind: 'option' },
+    { text: 'Motivated to stop or reduce', value: 'FULL_MOTIVATION', kind: 'option' },
+    { text: 'Unknown', value: 'UNKNOWN', kind: 'option' },
+  ],
+}
+
+export default {
+  drugsPractitionerAnaLysisMotivatedToStop,
```

Comment:
Ana Lysis? Isn't she an author?

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/pull/798/commits/a0fb6cfcbdd25e7057821300ae17968dff24a759